Jamfile

```diff
@@ -276,10 +273,12 @@ rule building ( properties * )
 
 	if <toolset>msvc in $(properties) || <toolset>intel-win in $(properties)
 	{
-		# allow larger .obj files (with more sections)
-		result += <cflags>/bigobj ;
 		# https://docs.microsoft.com/en-us/cpp/build/reference/utf-8-set-source-and-executable-character-sets-to-utf-8?view=msvc-170
 		result += <cflags>/utf-8 ;
+		# https://devblogs.microsoft.com/cppblog/msvc-now-correctly-reports-__cplusplus/
+		result += <cxxflags>/Zc:__cplusplus ;
+		# allow larger .obj files (with more sections)
+		result += <cflags>/bigobj ;
```

Comment:
the order here doesn't matter, right?

Reply:
the order is only for syncing between build scripts for easier maintenance, nothing else.